### PR TITLE
SAK-32329 Removed the conflict with the search tests

### PR DESCRIPTION
**Pull request by ddelblanco**

### CODE REVIEW

File: search/search-tool/tool/pom.xml [hunk 1]
```diff
@@ -100,6 +100,11 @@
         <artifactId>commons-lang3</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.elasticsearch</groupId>
```
Comment (jonespm) [inline search/search-tool/tool/pom.xml]: Is this dependency required? It doesn't look like it's used anywhere in the tool.

Comment (jonespm) [inline search/search-tool/tool/pom.xml]: Oh, I see for the test maybe? Maybe it can just have a 

`<scope>test</scope>`

**Comment by ddelblanco** [comment]
Yes scope test can be enough. I'll change in that way

File: search/search-tool/tool/pom.xml (hunk 1 above)
Comment (ddelblanco) [inline search/search-tool/tool/pom.xml]: Done!

**Comment by ern** [comment]
Update the commit message and add JIRA issue

**Comment by ddelblanco** [comment]
Done

**Pull request merged**